### DISCUSSION TITLE
스크린샷 랜더링 오류

### DISCUSSION THREAD

**Discussion by myunglark96** [naverz/zepeto-studio-kor#452] [Issue Report (Legacy)]
안녕하세요 

스크린샷 예제 프로젝트를 이용하여 스크린샷 기능을 구현했는데 player가 체형이 바뀌는 특정 아이템을(ex, 다리가 길어지는) 착용 할 시 스크린샷에 케릭터의 에셋들이 제대로 위치해 있지 않는 오류가 발생합니다.  
해당 에셋들을 찾아서 Root bones를 상위에 있는 오브젝트로(head -> hips) 바꾸어보니 정상 작동하는 것은 확인됐지만 케릭터를 로딩할 때 이러한 에셋들을 모두 찾아서 변경하는게 힘들거 같은데 혹시 해결 방법이 있을까요?

![캡처1](https://user-images.githubusercontent.com/53140732/156338922-a91d0f67-217f-4232-9931-ba79b0da2f53.PNG)
![캡처 2](https://user-images.githubusercontent.com/53140732/156338945-fefa2037-364c-4803-968c-bd0e75aeee91.PNG)
![머리 1](https://user-images.githubusercontent.com/53140732/156338956-7c12aa81-3a6c-4543-9621-9c57dfdf740d.PNG)
![머리2](https://user-images.githubusercontent.com/53140732/156338979-e3333b89-1d92-458a-bd76-b36e18a3856a.PNG)
![안경](https://user-images.githubusercontent.com/53140732/156338989-9a574b1f-d0ac-4610-ad20-e6bc54cbdbd2.PNG)
![안경2](https://user-images.githubusercontent.com/53140732/156338997-0b62b4bc-e989-4d85-a89e-49eb0ca7478d.PNG)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요~ 위의 스크린샷 오류 수정한 버전을 깃에 push해 두었습니다.

https://github.com/naverz/zepeto-multiplay-example

깃허브 최신 커밋내용을 참고해서 캡처 기능 구현 부탁드립니다.
감사합니다~
